src/gengowatcher/ui_textual.py

```
@@ -58,6 +60,80 @@ class Icons:
     POLLING = "↻"
 
 
+def _format_timestamp(timestamp: Any) -> str:
```

Ruff (0.14.14)

[error] Undefined name `Any`

(F821)